ui/src/app/job-details/resources/resources-table/resources-table.component.css

```diff
@@ -0,0 +1,3 @@
+.resource-button {
+  color: black;
```

Comment:
Background on this? I tried this change in Chrome inspector and I think I prefer having standard link styling.

Reply:
Switched back to standard.